zoo/repos/views.py

```diff
@@ -35,56 +22,3 @@ def repo_details(request, provider, repo_id):
         raise Http404(f"Project {repo_id} not found")
 
     return JsonResponse(details)
-
-
-def _parse_file(path, base=None):
```

Comment:
I think we should keep the git filesystem as fallback 🤔 
If the service has an url on the environment we'll load it, if not we'll fallback to check the git repo.

What do you think?

Reply:
hmm ok,and in case that there is an url on the environment and also repo... we will load the one from environment, right?

Reply:
yep, environment will always have predence.